census-rm-toolbox-deployment.yml

```diff
@@ -133,8 +133,8 @@
             secretName: read-replica-cloud-sql-certs
             defaultMode: 0400
             items:
-            - key: "root.crt"
-              path: "root.crt"
+            - key: "home/toolbox.crt"
+              path: "home/toolbox.crt"
```

Comment:
This should still be `root.crt` (the file is named that because it's the root certificate, unrelated to the root directory we were putting the files in)